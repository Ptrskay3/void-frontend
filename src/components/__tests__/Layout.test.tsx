import React from "react";
import { shallow } from "enzyme";
import { Layout } from "../Layout";

describe("Layout", () => {
  it("Basic small Layout renders", () => {
    const layout = shallow(<Layout variant="small" />);
    expect(layout.getElements()).toMatchSnapshot();
  });

  it("Basic regular Layout renders", () => {
    const layout = shallow(<Layout variant="regular" />);
    expect(layout.getElements()).toMatchSnapshot();
  });
});
