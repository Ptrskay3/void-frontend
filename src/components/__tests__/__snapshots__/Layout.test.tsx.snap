// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout Basic regular Layout renders 1`] = `
Array [
  <React.Fragment>
    <NavBar />
    <Wrapper
      variant="regular"
    />
  </React.Fragment>,
]
`;

exports[`Layout Basic small Layout renders 1`] = `
Array [
  <React.Fragment>
    <NavBar />
    <Wrapper
      variant="small"
    />
  </React.Fragment>,
]
`;
